.github/workflows/snyk.yml

```diff
@@ -0,0 +1,43 @@
+name: CI / snyk.io
+
+on:
+  push:
```

Comment:
was wondering - why on push and not on pull_request?

Reply:
We need this to be happening on each commit push.
This will give you the  answer:
https://stackoverflow.com/a/73297089

Reply:
ok, let me rephrase it, why do we need it on each commit push, not just when PR is opened? I think typically when developing something people have their branch and regularly push there, in setup like this snyk will be run every time, which seems like a lot of unnecessary runs really (I have the same feeling about our CIs setup)

Reply:
So why we run CIs every time we push but not when we open a PR? Let's say I opened a PR, and then there were many change reqs on it, and for addressing some, I introduced a package that had some critical issues, Snyk will not find that, same goes with the rest of CI steps. You can break the tests in future commits during development after you opened a PR, you can introduce linter issues, you can introduce code smells. That's why we run them on every commit. Doesn't seem unnecessary to me, that's the reason we have them.

Reply:
By `just when PR is opened` I meant any activity on the PR so it will cover what you are talking about...